app/state/user/index.ts

```diff
@@ -220,7 +220,10 @@ export const userSlice = createSlice({
   extraReducers: (builder) => {
     builder.addCase(redeemBond, (s, a) => {
       if (!s.balance) return;
-      s.balance.klima = safeAdd(s.balance.klima, a.payload.value);
+      s.balance.klima = safeAdd(
```

Comment:
When autostake is `true` we need to increment the `sklima` balance. Here you only increment `klima` balance